src/app/Caches/Create/ReviewCacheConfig.tsx

```diff
@@ -1,29 +1,71 @@
 import React, { useEffect, useState } from 'react';
-import { Flex, Form, FormGroup, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { Alert, AlertVariant, Flex, Form, FormGroup, Text, TextContent, TextVariants } from '@patternfly/react-core';
 import { CodeEditor } from '@patternfly/react-code-editor';
 import { useTranslation } from 'react-i18next';
 import { CacheConfigUtils } from '@services/cacheConfigUtils';
 import { useCreateCache } from '@app/services/createCacheHook';
+import LanguageToggleRadios from './LanguageToggleRadios';
+import { ConfigDownloadType } from '@services/infinispanRefData';
+import { ConsoleServices } from '@services/ConsoleServices';
 
-const ReviewCacheConfig = (props: { setReviewConfig: (string) => void }) => {
+const ReviewCacheConfig = (props: {
+  setReviewConfig: (string) => void;
+  setContentType: (string) => void;
+  contentType: 'json' | 'yaml' | 'xml';
+}) => {
   const { configuration } = useCreateCache();
   const { t } = useTranslation();
   const [config, setConfig] = useState(CacheConfigUtils.createCacheConfigFromData(configuration));
 
+  const [language, setLanguage] = useState(ConfigDownloadType.JSON);
```

Comment:
yeah we can use props.contentType, origianally i was using it so that we can call convertToAllFormat api with correct language and config. 
But we can remove language state and contentType can be used.